lib/logstash_pipeline_generator.rb

```diff
@@ -38,10 +39,14 @@ def generate_logstash_pipeline(mapping)
         mutations << { 'lowercase' => [lsf(affected_field)] }
       elsif 'to_array' == row[:format_action]
         array_fields << lsf(affected_field)
+      elsif ['parse_timestamp'].include?(row[:format_action])
+        dates << {
+            'match' => [ lsf(row[:source_field]), row[:timestamp_format] ]
+        }
```

Comment:
Here in order to be able to support the destination field ([target](https://www.elastic.co/guide/en/logstash/current/plugins-filters-date.html#plugins-filters-date-target) param), we should instead create one `date` filter per row, instead of one that includes all timestamps.

```
  date {
    match => ["[some_timestamp_field]", "UNIX_MS"]
    target => "@timestamp"
  }
  date {
    match => ["[some_other_timestamp]", "UNIX_MS"]
    target => "@timestamp"
  }
  date {
    match => ["[some_new_timestamp]", "UNIX"]
    target => "destination_timestamp"
  } 
```